### PR TITLE
Adding post resume test data

### LINKED ISSUE #57: Adding test data to the post
Adding test data to `posts` to run when running migrate command

### PR DESCRIPTION
**Pull request by MatanP12**
- Adding migrations file to add 2 resumes
- Adding migrations file to add 2 comments
- Adding migration files to add 4 rating
- Adding several CV to files

fix #57

image

### CODE REVIEW

**Review by TomerNewmanPrograms — DISMISSED** [review]
LGTM

File: posts/migrations/0006_add_resume_test_data.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+TEST_USERNAME1 = 'AlonShakaroff'
+TEST_USERNAME2 = 'YuliSuliman'
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('posts', '0005_poll'), ('users', '0003_add_users_test'),
+    ]
+
+    def generate_rersume_test_data(apps, schema_editor):
```
Comment (TomerNewmanPrograms) [inline posts/migrations/0006_add_resume_test_data.py:15]: Did you mean `resume` ?

**Review by TomerNewmanPrograms — CHANGES_REQUESTED** [review]

File: posts/migrations/0006_add_resume_test_data.py (hunk 1 above)
Comment (MatanP12) [reply posts/migrations/0006_add_resume_test_data.py]: You are right, Done 😄

**Review by TomerNewmanPrograms — DISMISSED** [review]
LGTM

**Review by yulisuliman — APPROVED** [review]
Looks good!

**Review by natifridman — APPROVED** [review]

**Review by yulisuliman — APPROVED** [review]

**Review by TomerNewmanPrograms — APPROVED** [review]

**Review by natifridman — APPROVED** [review]

**Pull request merged**